functions/pantheon-check-undeployed

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Include all of the functions that we need.
+FUNCTIONS_DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$FUNCTIONS_DIR" ]]; then FUNCTIONS_DIR="$PWD"; fi
+. "$FUNCTIONS_DIR/pantheon-script-colours"
+
+##
+# Checks that code is deployed.
+##
+check_deployed() {
+  local sitename=$1
+  local dest=$2
+  
+  echo -e "Checking for undeployed code in ${sitename}."
+  
+  #Check if previous commit was deployed
+  check=$(terminus env:code-log ${sitename}.${dest} --field=labels | sed -n -e '2{p;q}')
```

Comment:
This format for the `sed` command is not POSIX compliant.  On OSX I get an error
```
$ terminus env:code-log bcj.dev --field=labels | sed -n -e '2{p;q}'
sed: 1: "2{p;q}
": extra characters at the end of q command
```

Can you try something different?